### ISSUE TITLE
Question: How to visualize text updates while animation is paused?

### ISSUE THREAD

**Issue by jdobrev** [airbnb/lottie-web#2545]
I'm trying to use Lottie for animating Text Inputs with limited success. 

if I use 
`anim?.renderer.elements[2].updateDocumentData({ t: newText })`
during animation the animated text field will update properly. 

But if I do the same while the animation is paused, the updated text will refresh only once animation is unpaused. 

I couldn't find a way to refresh animation while paused. Is there an easy way to solve this issue?

**Comment by bodymovin** [comment]
yes, unfortunately that's a limitation of the player. I'd try to render a close subframe so the difference is visible.

**Comment by jdobrev** [comment]
Ye, this is how I did it in the end. Thanks!

**Comment by shinolex** [comment]
Hello o/ It's been a long time but I have the same issue. I update the text with updateDocumentData function but my animation is in pause state and doesn't get updated properly. Can you explain more of your workaround?
